eva-accession-release-automation/gather_clustering_counts/bash/count_rs_for_all_files.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -e
+
+OUTPUT_FILE=$1
+FILE_WITH_ALL_INPUTS=$2
+
+#Initialise list of tmp output
+ALL_TMP_OUTPUT=""
+
+# Uses all input as file to process
+for INPUT in `cat $FILE_WITH_ALL_INPUTS`
+do
+    echo "Process $INPUT"
+    ASSEMBLY=$(basename $(dirname ${INPUT}));
+    SC_NAME=$(basename $(dirname $(dirname ${INPUT})));
+    TYPE=$(echo $(basename ${INPUT}) | cut -f 4- -d '_' | awk '{print substr($0,1,length($0)-11)}')
+    OUTPUT=tmp_${SC_NAME}_${ASSEMBLY}_${TYPE}.txt
+    if [[ ${INPUT} == *.vcf.gz ]]
+    then
+        zcat  "${INPUT}" | grep -v '^#' | awk -v annotation="${ASSEMBLY}-${SC_NAME}-${TYPE}" '{print $3" "annotation}' > ${OUTPUT}
```

Comment:
**Optional:** Given the normalization and stability issues with scientific names, I am wondering if we should do a "reverse lookup" (from a static table or EVAPRO) to use the taxonomy instead of SC_NAME.

Reply:
I don't think that's necessary. The bash script deals with files and return annotation based on the filepath. As long as the file path reflect information we want to gather, we should be fine. We should deal with lookup in the python layer.

Reply:
Alternatively, we could also pass the annotation with the file name so that there is no guess work in the bash script.